package-lock.json

```diff
@@ -33,12 +33,12 @@
         "eslint-plugin-jsdoc": "^37.0.3",
         "eslint-plugin-node": "11.1.0",
         "eslint-plugin-prettier": "4.0.0",
-        "jest": "26.6.3",
+        "jest": "27.4.7",
         "jest-junit": "13.0.0",
         "prettier": "2.5.1",
         "semantic-release": "18.0.1",
         "sinon": "12.0.1",
-        "ts-jest": "26.5.6",
+        "ts-jest": "^27.1.2",
```

Comment:
@ricellis Have you removed the `node_modules` dir before the local `npm install`? I can see some differences in my lock file with this change, but idk whether it is a problem.

If I just checkout your changes, then `rm -rf node_modules` then `npm install` I get `27.1.2` without the `^`

Reply:
I thought I did, let me give it another prod

Reply:
I must have done the first time, but I guess not when I fixed the range. Rebased with the corrected lockfile.

Reply:
Thanks!